app/extensions.js

```diff
@@ -198,16 +198,19 @@ let generateBraveManifest = () => {
     'style-src': '\'self\' \'unsafe-inline\'',
     'font-src': '\'self\' data:',
     'img-src': '* data: file://*',
+    'connect-src': 'https://www.youtube.com https://www.twitch.tv',
```

Comment:
does twitch still need to be here?

Reply:
we don't need twitch to use this functionality

Reply:
in that case twitch can be removed from this line

Reply:
removed